Misc/NEWS.d/next/Tools-Demos/2020-05-03-01-30-46.bpo-36310.xDxxwY.rst

```diff
@@ -0,0 +1 @@
+Allow ``pygettext.py`` to detect calls to ``gettext`` in f-strings.
```

Comment:
```suggestion
Allow :file:`Tools/i18n/pygettext.py` to detect calls to ``gettext`` in
f-strings.```

Reply:
I committed this change but deleted the trailing triple backticks as I think you probably didn't intend to add them? Let me know if they were supposed to be there and I'm just missing something here.

Reply:
Yeah, an oversight.